### ISSUE TITLE
Fails to parse xml element with newline right after opening tag name.

### ISSUE THREAD

**Issue by evant** [Tickaroo/tikxml#55]
This is with `0.8.9-SNAPSHOT`

If you have the xml
```xml
<foo
></foo>
```
it will fail to parse with
```java
reader.beginElement();
reader.nextElementName();
reader.endElement(); // fails here
```
with
```
java.io.IOException: Expected a closing element tag </foo
> but found </foo> at path /foo
/text()

	at com.tickaroo.tikxml.XmlReader.syntaxError(XmlReader.java:850)
	at com.tickaroo.tikxml.XmlReader.doPeek(XmlReader.java:276)
	at com.tickaroo.tikxml.XmlReader.endElement(XmlReader.java:344)
```

It appears that it's incorrectly including a newline ('\n') char in the element tag name, causing it to fail to match the ending tag.

While the placement of the newline above may look odd, it's pretty common when attributes are included. Ex:
```xml
<foo
  bar1="baz1"
  bar2="baz2">
</foo>
```

**Comment by sockeqwe** [comment]
I see,
Thanks for reporting! Will fix that on Monday

**Comment by sockeqwe** [comment]
Fixed in latest version `0.8.10-SNAPSHOT`
